CHANGELOG.md

```diff
@@ -3,10 +3,12 @@
 ## Future Release
 
 - Updated Log in and Sign up form to match current styling
+- Fixed cursors to use the proper type
```

Comment:
Can we provide a stronger explanation of this? Where will people expect to find changes? Why were they wrong?

Something simple would probably go a long way…

```
Changed cursor over non-editable items to pointer
```

What were the cursors even before-hand?

Reply:
Something like this, maybe?

> Hovering over a clickable or editable UI element now show the correct cursor for its type